### ISSUE TITLE
Problema para setar as formas de entregas disponíveis

### ISSUE THREAD

**Issue by Vivapixel** [pedro-teixeira/correios#350]
Bom dia pessoal.

Temos um cliente que precisa setar apenas uma forma de envio para determinados produtos. Ou seja, tem produtos que ele só pode enviar por PAC, e tem produtos que ele só pode enviar por SEDEX.

No módulo, dentro do produto, temos ao opção "Serviços de Entrega" que permite selecionar os serviços apropriados para o produto.

Mas se selecionar APENAS PAC, quando faz o calculo do valor no checkout, ele exibi tanto o PAC quanto o SEDEX.

Isso seria um bug do módulo? Como posso restringir o tipo de envio por produto?

Desde já agradeço!
![screenshot_1](https://user-images.githubusercontent.com/11544673/38741059-6870239e-3f0f-11e8-9474-5a417b93cfcf.png)

Att

**Comment by rafaelpatro** [comment]
Tem que marcar nas configurações do módulo para Filtrar Serviços por Produto

**Comment by Vivapixel** [comment]
Obrigado Rafael! Deu certo. Mas, no checkout ele só mostra PAC, por mais que um produto só seja SEDEX e o outro só seja PAC! Não deveria mostrar no checkout as duas formas de entrega pré-selecionadas? Sabe me dizer como funciona isso nesse caso?

**Comment by rafaelpatro** [comment]
Sim. Também precisa manter selecionado nas configurações do módulo, todas
as formas de entrega q quer usar.

As formas de entrega selecionadas no produto só serão consideradas, se
estiverem marcadas nas configurações do módulo.

Sugestão: nas configurações do módulo deixe todos os serviços marcados.
Então nos produtos faça a personalização q quiser.

Em sex, 13 de abr de 2018 6:15 PM, Vivapixel <notifications@github.com>
escreveu:
